src/ca/index.ts

```diff
@@ -49,7 +49,11 @@ export class CAClient implements CA {
 
       return certificate.signedCertificateEmbeddedSct.chain.certificates;
     } catch (err) {
-      throw new InternalError('error creating signing certificate', err);
+      throw new InternalError({
+        code: 'CREATE_SIGNING_CERTIFICATE_ERROR',
```

Comment:
Should we have some pattern where we prefix the code with the module/thing?
```suggestion
        code: 'CA_CREATE_SIGNING_CERTIFICATE_ERROR',
```